src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -199,13 +203,11 @@ static void checkArg(string arg, string name)
             {
                 logger.LogIterationHeader("Profile Iteration");
                 ProfileParser profiler = new ProfileParser(parser);
-                using (var profileSession = TraceSessionManager.CreateSession("ProfileSession", "profile_"+traceName, traceDirectory, logger))
+                using var profileSession = TraceSessionManager.CreateSession("ProfileSession", "profile_" + traceName, traceDirectory, logger);
```

Comment:
Just curious, what do we benefit from removing the brackets here?

Reply:
It's a neat addition to C# 8.0, which allows you to create `using` declarations, which simplify the code and make it a bit more readable. It's particularly helpful when it helps you simplify multiple nested `using` code blocks:

```cs
using (DisposableType1 var1 = ...)
{
    using (DisposableType2 var2 = ...)
    {
        using (DisposableType3 var3 = ...)
        {
            ...
        }
    }
}
```

You can simplify it with:
```cs
using DisposableType1 var1 = ...;
using DisposableType2 var2 = ...;
using DisposableType3 var3 = ...;
...
```

You just need to keep in mind that you're now telling the compiler that the object(s) should get automatically disposed at the end of the current scope. There may be exceptions where you explicitly need to know when to dispose.

Here is the documentation: https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations